pnpbridge/src/adapters/src/mqtt_pnp/json_rpc.cpp

```diff
@@ -1,10 +1,11 @@
+#pragma once
+

```

Comment:
necessary on .cpp ?

Reply:
Not necessary, removing.

---
In reply to: [469535315](https://github.com/Azure/AzurePnPBridgePreview/pull/34#discussion_r469535315) [](ancestors = 469535315)